packages/unified-latex-to-pretext/libs/author-info.ts

```diff
@@ -0,0 +1,38 @@
+import * as Ast from "@unified-latex/unified-latex-types";
+import { visit } from "@unified-latex/unified-latex-util-visit";
+import { anyMacro } from "@unified-latex/unified-latex-util-match";
+import { htmlLike } from "@unified-latex/unified-latex-util-html-like";
+
+export function gatherAuthorInfo(ast: Ast.Ast): String[] {
+    const authorList: any[] = [];
+
+    visit(ast, (node) => {
+        if (anyMacro(node) && node.args) {
+            if (node.content == "author" || node.content == "address" || node.content == "email" ) {
```

Comment:
You should be using macro matchers for this `if (match.macro(node, "author")) { ... }`